libqtile/widget/mpdwidget.py

```diff
@@ -288,27 +271,26 @@ def poll(self):
         was_connected = self.connected
 
         if not self.connected:
-            if self.reconnect:
+            if self.connected is None or self.reconnect:
                 while not self.stop and not self.connect(quiet=True):
                     time.sleep(self.reconnect_interval)
             else:
-                return
+                return None
```

Comment:
I think someone has been cleaning up return statements like this to just be `return`, since `return None` is equivalent to `return`